catchup/peerSelector.go

```diff
@@ -259,10 +277,10 @@ func makePeerSelector(net peersRetriever, initialPeersClasses []peerClass) *peer
 	return selector
 }
 
-// GetNextPeer returns the next peer. It randomally selects a peer from a pool that has
+// getNextPeer returns the next peer. It randomally selects a peer from a pool that has
 // the lowest rank value. Given that the peers are grouped by their ranks, allow us to
 // prioritize peers based on their class and/or performance.
-func (ps *peerSelector) GetNextPeer() (peer network.Peer, err error) {
+func (ps *peerSelector) getNextPeer() (psp *peerSelectorPeer, err error) {
```

Comment:
I think that my preference here would to keep it with lower-case, as it's not being exported outside the scope of this package. ( we have plenty of components that do.. and it's a good marker for when we need to be more careful around interfaces )